low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -106,10 +110,16 @@ def read_data(self):
 
     def run(self):
         """Run the sensor on its own thread"""
-
+        DEFAULT_POLLING_INTERVAL = "604800"
```

Comment:
can you please add a comment mentioning what this number means? microseconds/seconds/etc...

Reply:
DEFAULT_POLLING_INTERVAL has been set as 604800 seconds.
As we need to check once a week, the integrity of raid data

Reply:
ok so the number is in seconds... i am requesting to add a comment in code to indicate that this number is in seconds.

Reply:
Done